support/runtime-upgrade/index.js

```diff
@@ -0,0 +1,57 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+
+var argv = require('minimist')(process.argv.slice(2));
+const fs = require('fs');
+
+// Create a promise API instance of the passed in node address.
+async function createPromiseApi(nodeAddress) {
+    const wsProvider = new WsProvider(nodeAddress);
+    const api = await new ApiPromise({ provider: wsProvider });
+    await api.isReady;
+    return api;
+}
+
+async function main() {
+    const nodeAddress = argv.nodeAddress;
+    const api = await createPromiseApi(nodeAddress);
+    const keyring = new Keyring({ type: 'sr25519' });
+    // Retrieve the upgrade key from the chain state
+    const adminId = await api.query.sudo.key();
+
+    // Some mnemonic phrase
+    const PHRASE = fs.readFileSync(argv.rootMnemonicPath).toString().trimEnd();
+
+    // Add an account, straight mnemonic
+    const newPair = keyring.addFromUri(PHRASE);
+
+    // Retrieve the runtime to upgrade
+    const code = fs.readFileSync(argv.wasmPath).toString('hex');
+    const proposal = api.tx.system && api.tx.system.setCode
+        ? api.tx.system.setCode(`0x${code}`) // For newer versions of Substrate
+        : api.tx.consensus.setCode(`0x${code}`); // For previous versions
```

Comment:
We don't run previous versions, we should only include the code for the version we run.